test/smallmatrix.jl

```diff
@@ -121,7 +121,54 @@ rmat = CrossRecurrenceMatrix(sparse(i,j,trues(length(i))))
     end
 end
 
-### Recurrence network
+### Skeletonization of RPs
```

Comment:
Could you please put these tests in a new test file?

Reply:
done